Cargo.toml

```diff
@@ -506,6 +506,10 @@ default-features = false
 [workspace.dependencies.smallvec]
 version = "1.14"
 
+[workspace.dependencies.tokio]
+version = "1"
+default-features = false
```

Comment:
nit: by default, there are no active features in `tokio`

Reply:
removed